main.scss

```diff
@@ -143,22 +136,12 @@
 	}
 
 	@if $martix {
```

Comment:
typo?

Reply:
YES!

Reply:
Which effects the public interface. Good catch